### PR TITLE
Implement API Level Tests

### PR DESCRIPTION
**Pull request by BarisSari**
This PR also includes some basic refactoring such as renaming course API module, removing enrolment related modules, removing prints, etc.

I thought it makes sense to clear unnecessary codes before starting `process enrolment` task. That's why I did the refactoring.

### CODE REVIEW

File: tests/api/test_courses.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    data = test_data.sample_create_course_request
+    response = client.post("/course", data=data.json())
+
+    assert response.status_code == FailureType.RESOURCE_ERROR.value
+    assert response.json() == {"detail": "RESOURCE_ERROR: " + message}
+
+
+@mock.patch("app.use_cases.update_course.UpdateCourse")
+def test_update_course_success(use_case):
+    message = "The callback has been fetched from the server."
```
Comment (TheAyeshaSiddiqa) [inline tests/api/test_courses.py:59]: "The course has been updated."

File: tests/api/test_courses.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    data = test_data.sample_update_course_request
+    response = client.put("/course", data=data.json())
+
+    assert response.status_code == FailureType.RESOURCE_ERROR.value
+    assert response.json() == {"detail": "RESOURCE_ERROR: " + message}
+
+
+@mock.patch("app.use_cases.get_course.GetCourseByID")
+def test_get_course_by_id_success(use_case):
+    message = "The callback has been fetched from the server."
```
Comment (TheAyeshaSiddiqa) [inline tests/api/test_courses.py:93]: "The course has been fetched."

**Review by TheAyeshaSiddiqa — APPROVED** [review]
minor changes.

File: tests/api/test_courses.py (hunk 1 above)
Comment (BarisSari) [reply tests/api/test_courses.py]: Done: https://github.com/ACWIC/training-provider-admin/pull/15/commits/c7b69122d93427ad04b01d4aa4a65247c8bfd379

File: tests/api/test_courses.py (hunk 2 above)
Comment (BarisSari) [reply tests/api/test_courses.py]: Done: https://github.com/ACWIC/training-provider-admin/pull/15/commits/c7b69122d93427ad04b01d4aa4a65247c8bfd379

**Review by TheAyeshaSiddiqa — APPROVED** [review]

**Pull request merged**